### PR TITLE
Info window button

### PR DESCRIPTION
**Pull request by nagmayasmin**
Info window with button

### CODE REVIEW

**Comment by gdhakal** [comment]
@nagmayasmin I can find two HTML files. Are these :two:  samples?

File: web-js/info-window-button/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+
+The sample has a button in the infoWindow which shows up
+when you click a point on the map.The clicked point is reverse geocode and shows the addrerss
```
Comment (jgravois) [inline web-js/info-window-button/README.md]: geocode**d**
~~addrerss~~ > *address*

File: web-js/esriSignInDialog [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+     or<br>
+     User Name: <b>user2</b><br>
+     Password: <b>user2</b></p>
+      <div id="content" style="height:100%;">
+        <div id="templatePickerDiv"></div>
+      </div>
+   </div>
+  </div>
+</body>
+</html>
```
Comment (jgravois) [inline web-js/esriSignInDialog]: did you mean to commit this file?

File: web-js/info-window-button/infoWindow_button.html [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+          infoTemplate.setTitle("Location");
+          infoTemplate.setContent("<b>Address :</b>${Address}<br/><br/><button id='LabelButton' onclick='myFunction();'>Label it</button>"); 
+		  
+		  locator.on("location-to-address-complete", function (evt) {
+              if (evt.address.address) {
+                  locator.outSpatialReference = map.spatialReference;  //// *** THIS
+                  address = evt.address.address;
+                  location = evt.address.location;  //// *** THIS
+          				 
+				// graphic_org = new Graphic(location, symbol, address, infoTemplate);
```
Comment (jgravois) [inline web-js/info-window-button/infoWindow_button.html]: it'd be good to get rid of all the non-informative comments and add a few additional comments to explain the code flow a bit.

**Comment by jgravois** [comment]
thanks @nagmayasmin!

**Pull request merged**